health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -103,26 +102,29 @@ export const UICustomizations = {
 
     customValidationCheck: (data) => {
       const { phone } = data;
+      debugger
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)